tests/integration/helpers/cluster.py

```diff
@@ -4405,10 +4405,7 @@ def write_embedded_config(name, dest_dir, fix_log_level=False):
             )
 
         write_embedded_config("0_common_instance_users.xml", users_d_dir)
-        if (
-            os.environ.get("CLICKHOUSE_USE_NEW_ANALYZER") is not None
```

Comment:
Now old analyzer integration tests simply checks new analyzer instead, because it had been broken in 67d39a22acbb91ed5, let's try enable it back - https://github.com/ClickHouse/ClickHouse/pull/63069